controller/class.tapestry-controller.php

```diff
@@ -10,18 +10,37 @@ class TapestryController
         'TAPESTRY' => 'tapestry',
         'TAPESTRY_NODE' => 'tapestry_node'
     ];
+    const NODE_PERMISSIONS = [
+        'DEFAULT' => [
+            'public' => 'read'
+        ],
+        'OPTIONS' => [
+            'READ_NODE' => 'read',
+            'EDIT_SUBMIT_NODE' => 'edit_submit',
+            'ADD_SUBMIT_NODE' => 'add_submit',
+            'DELETE_NODE' => 'delete'
```

Comment:
Please add `'edit'`, `'add'`, and `'approve'` permissions. `'delete'` permission should be removed. Anyone who has `'edit'` permission should be also able to delete. `'edit_submit'` should be able to submit deletion that goes through a similar approval process.